libPhoneNumber-GeocodingParser/libPhoneNumber-GeocodingParser/NBGeocoderMetadataParser.m

```diff
@@ -0,0 +1,74 @@
+//
+//  NBGeocoderMetadataParser.m
+//  libPhoneNumber-GeocodingParser
+//
+//  Created by Rastaar Haghi on 7/1/20.
+//  Copyright © 2020 Google LLC. All rights reserved.
+//
+
+#import "NBGeocoderMetadataParser.h"
+
+@implementation NBGeocoderMetadataParser {
+@private
+  SQLiteDatabaseConnection *_databaseConnection;
+}
+
+- (instancetype)init {
+  self = [super init];
+  return self;
+}
```

Comment:
You can remove this since you're not doing anything special here.

Reply:
done.